app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,27 @@
+class MessagesController < ApplicationController
```

Comment:
can we add some `before_action`s in here to ensure that only an admin can create a message?